Sergey_tasks/JS/Bank-project/Bank.js

```diff
@@ -0,0 +1,275 @@
+
+// BANKIST APP
+const account1 = {
+  owner: 'Jonas Schmedtmann',
+  movements: [],
+  pin: 1,
+  login: "js",
+  currentBalance: 1000,
+};
+
+const account2 = {
+  owner: 'Jessica Davis',
+  movements: [],
+  pin: 2,
+  login: "jd",
+  currentBalance: 2000,
+};
+
+const account3 = {
+  owner: 'Steven Thomas Williams',
+  movements: [],
+  pin: 3,
+  login: "stw",
+  currentBalance: 3000,
+};
+
+const account4 = {
+  owner: 'Sarah Smith',
+  movements: [],
+  pin: 4,
+  login: "ss",
+  currentBalance: 4000,
+};
+let accounts = [account1, account2, account3, account4];
+
+// Elements
+const labelWelcome = document.querySelector('.welcome');
+const labelDate = document.querySelector('.date');
+const labelBalance = document.querySelector('.balance__value');
+const labelSumIn = document.querySelector('.summary__value--in');
+const labelSumOut = document.querySelector('.summary__value--out');
+const labelSumInterest = document.querySelector('.summary__value--interest');
+const labelTimer = document.querySelector('.timer');
+//---------------------------------------------------------------------------
+const containerApp = document.querySelector('.app');
+const containerMovements = document.querySelector('.movements');
+const movementsRowDeposit = document.querySelector(".movements__row-deposit");
+const movementsRowWithdrawal = document.querySelector(".movements__row-withdrawal");
+const movementsTypeDeposit = document.querySelector(".movements__type-deposit");
+const movementsTypeWithdrawal = document.querySelector(".movements__type-withdrawal");
+const movementsValueDeposit = document.querySelector(".movements__value-deposit");
+const movementsValueWithdrawal = document.querySelector(".movements__value-withdrawal");
+//---------------------------------------------------------------------------
+const btnLogin = document.querySelector('.login__btn');
+const btnTransfer = document.querySelector('.form__btn--transfer');
+const btnLoan = document.querySelector('.form__btn--loan');
+const btnClose = document.querySelector('.form__btn--close');
+const btnSort = document.querySelector('.btn--sort');
+//---------------------------------------------------------------------------
+const inputLoginUsername = document.querySelector('.login__input--user');
+const inputLoginPin = document.querySelector('.login__input--pin');
+const inputTransferTo = document.querySelector('.form__input--to');
+const inputTransferAmount = document.querySelector('.form__input--amount');
+const inputLoanAmount = document.querySelector('.form__input--loan-amount');
+const inputCloseUserName = document.querySelector('.form__input--user');
+const inputClosePin = document.querySelector('.form__input--pin');
+
+//--Переменные ---------------------------------------------------------------
+const today = new Date();
+let closeActiveUser;
+
+//--Запись в данных массива accounts в localStorage если они не записаны
+function init(){
+   if(JSON.parse(localStorage.getItem("balance")))
```

Comment:
we can simplify this code by updating if statement
```suggestion
   if(!JSON.parse(localStorage.getItem("balance"))) {
        return
   }
```